app/fullService/axiosFullService.ts

```diff
@@ -59,12 +60,10 @@ const axiosFullService = async <T>(
 
     // TODO: determine if we need to handle errors here or elsewhere
     // such as the API or services
-    return skipKeysCamelCase(response);
+    return camelCaseObjectKeys(response);
```

Comment:
skipKeysCamelCase is a package that has been camelCasing ids that we've been using as keys. This caused a lot of issues where lookup patterns of `objectWithCamelCasedIdKeys[lowerCaseId]` would return nothing, even when it should have been a valid lookup.